docs/gitbook/docker/getting_started.md

```diff
@@ -17,29 +17,31 @@
   under the License.
 -->
 
+# Getting started with Hivemall on Docker
+
 This page introduces how to run Hivemall on Docker.
 
 <!-- toc -->
 
 > #### Caution
 > This docker image contains a single-node Hadoop enviroment for evaluating Hivemall. Not suited for production uses.
 
-# Requirements
+## Requirements
 
  * Docker Engine 1.6+
  * Docker Compose 1.10+
 
-# 1. Build image
+## 1. Build image
```

Comment:
Could you remove `1.` and `2.`?

See what's happing in
http://hivemall.incubator.apache.org/userguide/docker/getting_started.html#1-build-image

Reply:
Removed 8d6faa6